index.js

```diff
@@ -49,6 +49,8 @@ class Analytics {
     this.queue = []
     this.writeKey = writeKey
     this.host = removeSlash(options.host || 'https://api.segment.io')
+    this.retries = options.retries || 3
```

Comment:
"retries" sounds like boolean to me. Maybe something a bit more descriptive like "retryCount" would be better?

/cc @stevenmiller888